marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/models/getPrimaryEntityTypes.sjs

```diff
@@ -19,7 +19,20 @@ xdmp.securityAssert("http://marklogic.com/data-hub/privileges/read-entity-model"
 
 const entityLib = require("/data-hub/5/impl/entity-lib.sjs");
 
-fn.collection(entityLib.getModelCollection()).toArray().map(model => {
+let modelArr = fn.collection(entityLib.getModelCollection()).toArray();
+modelArr.sort(function(modelA, modelB) {
+  var nameA = modelA.toObject().info.title;
```

Comment:
I think it'd be better to sort these after they've been populated with "entityName". That minimizes the amount of code we have with knowledge of "info.title" meaning something important. 

So build up the array, then sort it by entityName.